src/back/routes/proposal.ts

```diff
@@ -131,7 +131,7 @@ export default routes((route) => {
   route.post('/proposals/hiring', withAuth, handleAPI(createProposalHiring))
   route.get('/proposals/priority/:address?', handleJSON(getPriorityProposals))
   route.get('/proposals/grants/:address', handleAPI(getGrantsByUser))
-  route.get('/proposals/:proposal', handleAPI(getProposal))
+  route.get('/proposals/:proposal', handleAPI(getProposalWithProject))
```

Comment:
I would not rename this function since these new attributes belong to the proposal